test/conformance/README.md

```diff
@@ -8,4 +8,4 @@ In the future, when all bugs are fixed, and the tests pass,
 this solution will no longer be necessary.
 When you fix any test, the match file must be updated
 Empty match files indicate that there are no failing tests
-in a particular group for the corresponding adapter.
+in a particular group for the corresponding adapter.
```

Comment:
is there any relevant change here? (perhaps GH is not showing me something)

Reply:
space at the end of sentence.